compiler/rustc_lint/src/unused.rs

```diff
@@ -1154,7 +1154,7 @@ impl EarlyLintPass for UnusedParens {
 
     fn check_stmt(&mut self, cx: &EarlyContext<'_>, s: &ast::Stmt) {
         if let StmtKind::Local(ref local) = s.kind {
-            self.check_unused_parens_pat(cx, &local.pat, true, false, (false, false));
+            self.check_unused_parens_pat(cx, &local.pat, true, false, (true, false));
```

Comment:
these awful unreadable arguments mean this:
```rust
        avoid_or: bool,
        avoid_mut: bool,
        keep_space: (bool, bool),
```

Reply:
@Nilstrieb do you want me to refactor this function's signature to make the args more readable?

Reply:
that would be nice as a follow up PR if you want to :3

Reply:
Okay, sure :)._